docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -138,10 +138,10 @@
         Administrator. After a configurable period of inactivity your ECL
         Watch session locks and you need to enter your credentials to unlock
         and resume your session. The session remains active for as long as
-        there is regular interaction.</para>
+        there is regular keyboard and mouse interaction.</para>
 
         <para>Additionally, there is a Logout menu item available you can
-        close your session when you complete your work.</para>
+        click on and close your session when you complete your work.</para>
```

Comment:
Additionally, there is a Logout menu item available you can select to close your session when you complete your work